Header header
string fixDate
string fixTime
float64 latitude
float64 longitude
float64 altitude
int32 fixQuality
int32 satellites
float64[6] standarDevs
float64 age
float64 ratio

